### PR TITLE
New character info screen

### PR DESCRIPTION
**Pull request by AngieDutra**
## Motivation

The character info screen was outdated

## Summary of changes

- deleted old character selection screen
- applied new styles and functionalities

reference (7)

## Checklist
- [ ] I have tested the changes locally.
- [ ] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

File: client/Assets/Scripts/UI/CharacterInfoManager.cs [hunk 1]
```diff
@@ -97,7 +56,8 @@ public void SetCharacterInfo(CoMCharacter comCharacter)
         classImage.sprite = comCharacter.classImage;
         skillDescriptions[0].SetSkillDescription(comCharacter.skillsInfo[0]);
         skillDescriptions[1].SetSkillDescription(comCharacter.skillsInfo[1]);
-        StartCoroutine(ModelManager.GetComponentInChildren<RotateUIModel>().GetModel());
+        skillDescriptions[2].SetSkillDescription(comCharacter.skillsInfo[2]);
+        StartCoroutine(rotateUIModel.GetModel());
```
Comment (BertovDev) [inline client/Assets/Scripts/UI/CharacterInfoManager.cs:60]: This is not necessary you are already calling it the Start method of `RotateUIModel` Script..

File: client/Assets/Scripts/UI/CharacterInfoManager.cs [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-using UnityEngine.Networking;
 using UnityEngine.UI;
 
 public class CharacterInfoManager : MonoBehaviour
 {
     [SerializeField]
     private UIModelManager ModelManager;
 
+    [SerializeField]
+    private RotateUIModel rotateUIModel;
```
Comment (BertovDev) [inline client/Assets/Scripts/UI/CharacterInfoManager.cs:14]: Same for this. We can remove it.

**Review by BertovDev — CHANGES_REQUESTED** [review]
Nice and just a few comments. Select button is not working due the missing component I mencioned above.

File: client/Assets/Scenes/CharacterInfo.unity
[the remark this answers is not in the document]
Comment (AngieDutra) [reply client/Assets/Scenes/CharacterInfo.unity:1]: Captura de pantalla 2024-02-05 a la(s) 15 36 52

I'm actually fixing the scene itself. It was broken in main. If things were working properly I should be able to checkout from main this scene and have all working properly. Instead this is the result. Therefore, these changes are vital

**Review by lotuuu — APPROVED** [review]
Looks good to me! Tablet resolution is a bit weird but I'm guessing that's not on our priority list right now. All phone resolutions looked ok
Screen Shot 2024-02-06 at 16 10 18

**Pull request merged**